VogtPayroll4/Program.cs

```diff
@@ -1,12 +1,21 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace VogtPayroll4
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            Payroll payroll = new Payroll();
+            List<Employee> empList = new List<Employee>();
+            empList = payroll.ReadEmployees();
```

Comment:
You should be calling the ConsoleReader to read the list of employees, then with the list calling payroll.AddEmployees(). Finally, then you can payroll.PrintAllEmployeeInfo()